tests/test_utils.py

```diff
@@ -52,22 +53,57 @@ def test_read_bed_file_good_input(self, tmpdir):
             fh.write("\n".join(bed_lines))
         assert utils.read_bed_file(bed_file) == expected_sites
 
+    def test_read_bed_file_empty_input(self, tmpdir):
+        """read_byd_file should return an empty list given an empty file"""
+        bed_file = str(tmpdir / "temp.bed")
+        expected_sites = []
+        with open(bed_file, "w") as fh:
+            fh.write("")
+        assert utils.read_bed_file(bed_file) == expected_sites
+
     def test_read_bed_file_with_header(self, tmpdir):
         """read_bed_file should skip header lines if they exist in bed files"""
         bed_file = str(tmpdir / "temp.bed")
-        bed_lines = ["CHROM\tSTART\tEND","SEQ\t7\t8", "SEQ\t2\t5"]
+        bed_lines = ["CHROM\tSTART\tEND","browser yaddayadda","track moreyadda","SEQ\t7\t8", "SEQ\t2\t5"]
+        expected_sites = [2,3,4,7]
+        with open(bed_file, "w") as fh:
+            fh.write("\n".join(bed_lines))
+        assert utils.read_bed_file(bed_file) == expected_sites
+
+    def test_read_bed_file_with_only_header(self, tmpdir):
+        """read_bed_file should skip header lines if they exist in bed files"""
+        bed_file = str(tmpdir / "temp.bed")
+        bed_lines = ["CHROM\tSTART\tEND"]
+        expected_sites = []
+        with open(bed_file, "w") as fh:
+            fh.write("\n".join(bed_lines))
+        assert utils.read_bed_file(bed_file) == expected_sites
+
+    def test_read_bed_file_with_internal_comment_line(self, tmpdir):
+        """read_bed_file should error out if any other lines are unreadable"""
```

Comment:
Suggestion: update the test function docstring

```suggestion
        """read_bed_file should ignore internal comment lines"""
```